models/roles/student.py

```diff
@@ -16,3 +16,11 @@ class Student(AbstractStudent):
         ],
         unique=True,
     )
+
+    def save(self, *args, **kwargs):
+        """
+        Override save method of Student class to automatically run validators of a model.
+        """
+
+        self.full_clean()
```

Comment:
Wrap this inside an `if` condition.

```python
if (kwargs.get('run_validations', False)):
    self.full_clean()
```

Reply:
@dhruvkb It is not working when I want to create an object. Works only when I explicitly called `save`